internal/controller/instaslice_controller.go

```diff
@@ -596,10 +625,12 @@ func (r *InstasliceReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		return err
 	}
 
-	return ctrl.NewControllerManagedBy(mgr).
+	controllerManager := ctrl.NewControllerManagedBy(mgr).
```

Comment:
do we need to assign this to variable?

Reply:
We don't need to currently, both versions achieve the same thing but I added it to make it easier to debug to inspect controllerManager initially. I reverted it to the original.